fxdc/writedata/parseobject.py

```
@@ -13,16 +13,19 @@ def __init__(self, data: object) -> None:
         Returns:
             str: Returns the string from the object
         """
         type_ = Config.get_class_name(data.__class__)
+        class_ = None
+        descriptions: dict[str, str] = {}
         debug(type_)
         try:
-            convertedobject = getattr(Config, type_).return_data(data)
+            class_ = getattr(Config, type_)
+            convertedobject, descriptions = class_.return_data(data)
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)